### PR TITLE
Query implementation

### PR DESCRIPTION
**Pull request by omersolmazer**
Query implemented with the procedure defined in the issue. 
Removed dict_inventory file
Get all method changed to return a list of products for consistency.

### CODE REVIEW

**Review by p2hang — COMMENTED** [review]
Looks good we just need one more change I think!

File: redis_inventory.py [hunk 1]
```diff
@@ -25,10 +26,10 @@ def get_product(self, product_id):
 
 
   def get_all(self):
-    result = {}
+    result = []
     for key in self.redis.keys():
       if key != 'index':  # filer out our id index
-        result[key] = self.redis.hgetall(key)
+        result.append(self.redis.hgetall(key))
```
Comment (p2hang) [inline redis_inventory.py:32]: I think we also need to add **product_id** to the **info** in **create_product()**

Or there will not be the product_id field.

**Comment by p2hang** [comment]
And let's remove the product.py in this cl too!

File: redis_inventory.py (hunk 1 above)
Comment (omersolmazer) [reply redis_inventory.py:32]: I've added that in the issue for data reformatting, its on its way right now

**Pull request merged**